blocks/disclaimer-modal/disclaimer-modal.js

```diff
@@ -0,0 +1,35 @@
+import { readBlockConfig } from '../../scripts/lib-franklin.js';
+
+export default async function decorate(block) {
+  const response = await fetch('/drafts/teriyana/disclaimer-modal.plain.html');
+  const isModalAccepted = document.cookie.match(/\shcpModalDismiss=1;?/) !== null || window.location.href.indexOf('?bypassModal') > -1;
+  const shouldShowModal = !isModalAccepted || (document.location.href.indexOf('?showModal') > -1);
```

Comment:
Why the URL parameter for forcibly showing the modal?

Reply:
For testing purposes, if you accept the cookie the while testing, the modal won't show up anymore. So this is just to override it to show the modal if needed.